Formula/aurora-cli.rb

```diff
@@ -11,9 +11,15 @@ class AuroraCli < Formula
     sha256 "62e565f0d65601f88b11d8c5f5bfad89d2f1661144072876fc705e50b4284329" => :yosemite
   end
 
+  patch do
+    url "https://raw.githubusercontent.com/thinker0/aurora/pants-fix-high-sierra/pants_version_1.4.0_dev20.diff"
```

Comment:
This will need to go in a PR to https://github.com/Homebrew/formula-patches

Reply:
@ilovezfs Thanks 

https://github.com/Homebrew/formula-patches/pull/206